pkg/asset/manifests/openshift.go

```diff
@@ -258,6 +259,38 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
 		assetData["99_private-cluster-outbound-service.yaml"] = applyTemplateData(privateClusterOutbound.Files()[0].Data, templateData)
 	}
 
+	if installConfig.Config.Azure != nil && installConfig.Config.Azure.IsARO() {
```

Comment:
nit: it looks like the other conditionals are switching on `platform` so perhaps for consistency it might be better to do:

```go
if platform == azuretypes.Name && installConfig.Config.Azure.IsARO()
```